packages/config/src/bridges/index.ts

```diff
@@ -19,6 +19,8 @@ import { wormholeBridge } from '../bridges/wormholeBridge'
 import { xDaiBridge } from '../bridges/xDaiBridge'
 import { BridgeDescription } from './types/bridge'
 
+export * from './types/bridge'
```

Comment:
maybe create index.ts inside /types ? I know it is not needed now but can be useful in the future